src/main/resources/com/nirima/jenkins/plugins/docker/DockerCloud/help-name.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Choose a name for this Docker cloud provider
```

Comment:
"Choose" because the user needs to provide a name, it is not a name already used at for example Docker Host configuration. I changed it to: " Provide a name for this Docker Cloud".